### DISCUSSION TITLE
License for family?

### DISCUSSION THREAD

**Discussion by pictosun** [rxhanson/Multitouch-Community#145] [Q&A]
Hi,

concerning the license. My wife also got a MacBook. Do I have to purchase the license again, or is it possible to install the license on more than one?
Also thinking about to install it on my second mac.

So at all we need 3 machines. This will then become expensive at some point.

And don't get me wrong - I like to pay for software - especially for such good stuff.

**Comment by rxhanson** [comment]
License keys can be activated on three devices at once, so you're all set with one license key.

Here are the details about transferring a license key in the event you get rid of a Mac or wipe macOS: https://github.com/rxhanson/Multitouch-Community/wiki/Transferring-Multitouch-Settings-to-a-Different-Mac

**Reply by pictosun** [reply]
Thx. a Lot. Didn't find those three license info.

**Reply by rxhanson** [reply]
It's on the website, but kind of obscure in the footer. I'll see if I can make it a little more prominent (you're not the first with this question).
